geodesy/package.xml

```diff
@@ -25,17 +26,24 @@
 
   <run_depend>geographic_msgs</run_depend>
   <run_depend>geometry_msgs</run_depend>
-  <run_depend>python-pyproj</run_depend>
   <run_depend>tf</run_depend>
   <run_depend>sensor_msgs</run_depend>
   <run_depend>unique_id</run_depend>
   <run_depend>uuid_msgs</run_depend>
 
+  <run_depend condition="$ROS_PYTHON_VERSION == 2">python-pyproj</run_depend>
```

Comment:
The `package.xml` will need to use `format="3"` to use conditional dependencies, which means `run_depend` will need to be changed to `exec_depend`.

Reply:
Will do.